Makefile

```diff
@@ -155,21 +154,17 @@ ensure-generated:
 test-e2e:
 	@test/e2e.sh
 
-HAS_DEP := $(shell $(CHECK) dep)
 HAS_GOX := $(shell $(CHECK) gox)
 HAS_GIT := $(shell $(CHECK) git)
 HAS_GOLANGCI ?= $(shell $(CHECK) golangci-lint)
 HAS_GINKGO := $(shell $(CHECK) ginkgo)
 
 .PHONY: bootstrap
 bootstrap:
-ifndef HAS_DEP
-	go get -u github.com/golang/dep/cmd/dep
-endif
 ifndef HAS_GOX
 	go get -u github.com/mitchellh/gox
 endif
-	go install ./vendor/github.com/go-bindata/go-bindata/...
+	go get github.com/go-bindata/go-bindata
```

Comment:
We installed `go-bindata` from our vendored directory to ensure we used a specific version, IIRC. Is there a way we can still do that? Ideally v3.1.2 since we have that vendored in `go.mod`.

Reply:
We actually specify the version in the go.mod file.

Reply:
Oh, **right**–`go get` is mod-aware now. Excellent!